sw/airborne/modules/rot_wing_drone/rot_wing_automation.c

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (C) 2023 Dennis van Wijngaarden <D.C.vanWijngaarden@tudelft.nl>
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/** @file "modules/rot_wing_drone/rot_wing_automation.c"
+ * @author Dennis van Wijngaarden <D.C.vanWijngaarden@tudelft.nl>
+ * Fucntions to automate the navigation and guidance of the rotating wing drone
+ */
+
+#include "modules/rot_wing_drone/rot_wing_automation.h"
+#include "state.h"
+#include "modules/datalink/telemetry.h"
+#include "filters/low_pass_filter.h"
+#include "state.h"
+
+#include "math/pprz_algebra_float.h"
+#include "modules/nav/waypoints.h"
+
+#ifndef ROT_WING_AUTOMATION_TRANS_ACCEL
+#define ROT_WING_AUTOMATION_TRANS_ACCEL 1.0
```

Comment:
All these defines are not documented both there and in the module file

Reply:
done